components/collective-page/sections/Contribute.js

```diff
@@ -210,7 +224,10 @@ class SectionContribute extends React.PureComponent {
     const isProject = collective.type === CollectiveType.PROJECT;
     const isFund = collective.type === CollectiveType.FUND;
     const hasCustomContribution = !collective.settings?.disableCustomContributions;
-    const hasContribute = isAdmin || (collective.isActive && (sortedTiers.length || hasCustomContribution));
+    const hasCryptoContribution =
+      !get(collective, 'settings.disableCryptoContributions', true) && collective?.host?.settings?.cryptoEnabled;
```

Comment:
Consistency, use the `get` used elsewhere to fetch `collective?.host?.settings?`

Reply:
Done, and I've also changed line 226 above this to correspond with the `get` syntax.